rules/docker-orchagent.mk

```diff
@@ -39,5 +39,5 @@ $(DOCKER_ORCHAGENT)_RUN_OPT += -v /var/log/swss:/var/log/swss:rw
 $(DOCKER_ORCHAGENT)_BASE_IMAGE_FILES += swssloglevel:/usr/bin/swssloglevel
 $(DOCKER_ORCHAGENT)_FILES += $(ARP_UPDATE_SCRIPT) $(ARP_UPDATE_VARS_TEMPLATE) $(SUPERVISOR_PROC_EXIT_LISTENER_SCRIPT)
 
-SONIC_BUSTER_DOCKERS += $(DOCKER_ORCHAGENT)
-SONIC_BUSTER_DBG_DOCKERS += $(DOCKER_ORCHAGENT_DBG)
+SONIC_BULLSEYE_DOCKERS += $(DOCKER_ORCHAGENT)
```

Comment:
`SONIC_BULLSEYE_DOCKERS` and `SONIC_BULLSEYE_DBG_DOCKERS` don't exist and aren't used, so lines adding to these variables can be removed.